codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/BoringSSLCertificateCallback.java

```diff
@@ -66,7 +66,7 @@ final class BoringSSLCertificateCallback {
     static final String KEY_TYPE_EC_RSA = "EC_RSA";
 
     // key type mappings for types.
-    private static final Map<String, String> KEY_TYPES = new HashMap<String, String>();
+    public static Map<String, String> KEY_TYPES = new HashMap<String, String>();
```

Comment:
why this change ?

Reply:
Same as previous.  Except this setting is for allowed client certificate types. The previous variable is for allowed server certificate types.

Allowing the user to add or remove from this list adds additional flexibility to this quic library. In my case, I needed to be able to use ed25519 certificates. I can now do so, without effecting other users who will keep the default settings.

Reply:
I think we should just do this via the constructor and so via a `BoringSslContextOption`

Reply:
Reverted this change in favour of 2 new ssl context options